its/ruling/src/test/resources/expected/python-S2638.json

```diff
@@ -1,311 +1,325 @@
+"project:biopython/Bio/SeqIO/SffIO.py": [
 1167
 ],
-'project:biopython/Bio/SeqIO/XdnaIO.py': [
+"project:biopython/Bio/SeqIO/XdnaIO.py": [
 233
 ],
-'project:biopython/Bio/codonalign/codonseq.py': [
+"project:biopython/Bio/codonalign/codonseq.py": [
 159
 ],
-'project:biopython/Bio/motifs/__init__.py': [
+"project:biopython/Bio/motifs/__init__.py": [
 212
 ],
-'project:buildbot-0.8.6p1/buildbot/pbutil.py': [
+"project:buildbot-0.8.6p1/buildbot/pbutil.py": [
 75
 ],
-'project:buildbot-0.8.6p1/buildbot/util/__init__.py': [
+"project:buildbot-0.8.6p1/buildbot/util/__init__.py": [
 166
 ],
-'project:buildbot-slave-0.8.6p1/buildslave/pbutil.py': [
+"project:buildbot-slave-0.8.6p1/buildslave/pbutil.py": [
 63
 ],
-'project:django-2.2.3/django/contrib/gis/db/backends/oracle/operations.py': [
+"project:django-2.2.3/django/contrib/gis/db/backends/oracle/operations.py": [
 117
 ],
-'project:django-2.2.3/django/contrib/gis/db/backends/postgis/operations.py': [
+"project:django-2.2.3/django/contrib/gis/db/backends/postgis/operations.py": [
 195,
 208
 ],
-'project:django-2.2.3/django/contrib/gis/db/backends/spatialite/operations.py': [
+"project:django-2.2.3/django/contrib/gis/db/backends/spatialite/operations.py": [
 105
 ],
-'project:django-2.2.3/django/core/management/commands/loaddata.py': [
+"project:django-2.2.3/django/core/management/commands/loaddata.py": [
 346
 ],
-'project:django-2.2.3/django/db/models/sql/compiler.py': [
+"project:django-2.2.3/django/db/models/sql/compiler.py": [
 1265,
 1327,
 1349,
 1365,
 1422,
 1487
 ],
-'project:django-2.2.3/django/test/runner.py': [
+"project:django-2.2.3/django/test/runner.py": [
 340
 ],
-'project:django-2.2.3/django/utils/archive.py': [
+"project:django-2.2.3/django/utils/archive.py": [
 151,
 193
 ],
-'project:django-2.2.3/django/utils/text.py': [
+"project:django-2.2.3/django/utils/text.py": [
 284
 ],
-'project:django-cms-3.7.1/cms/tests/test_menu.py': [
+"project:django-cms-3.7.1/cms/tests/test_menu.py": [
 1689
 ],
-'project:django-cms-3.7.1/cms/tests/test_menu_page_viewperm.py': [
+"project:django-cms-3.7.1/cms/tests/test_menu_page_viewperm.py": [
 258
 ],
-'project:django-cms-3.7.1/cms/tests/test_permmod.py': [
+"project:django-cms-3.7.1/cms/tests/test_permmod.py": [
 665
 ],
-'project:mypy-0.782/mypy/semanal.py': [
+"project:mypy-0.782/mypy/semanal.py": [
 4682,
 4749
 ],
-'project:mypy-0.782/mypy/typeanal.py': [
+"project:mypy-0.782/mypy/typeanal.py": [
 885
 ],
-'project:numpy-1.16.4/numpy/distutils/command/build_ext.py': [
+"project:numpy-1.16.4/numpy/distutils/command/build_ext.py": [
 284
 ],
-'project:numpy-1.16.4/numpy/distutils/fcompiler/pg.py': [
+"project:numpy-1.16.4/numpy/distutils/fcompiler/pg.py": [
 131
 ],
-'project:numpy-1.16.4/numpy/distutils/mingw32ccompiler.py': [
+"project:numpy-1.16.4/numpy/distutils/mingw32ccompiler.py": [
 176,
 177,
 178,
 179
 ],
-'project:numpy-1.16.4/numpy/ma/core.py': [
+"project:numpy-1.16.4/numpy/ma/core.py": [
```

Comment:
I'm a bit uneasy about accepting these as issues.
I have the feeling this rule could have an exception for "private" components (given away by the `_` in their names), as it could be argued that there is no broken contract if the parent is not meant to be used as-is (it might be challenged though).

If you agree, I can create a ticket to make this exception. 

For the time being though, I'd accept these issues as they document an existing behavior and have the special situation of appearing while analyzing `numpy` using stubs for `numpy`, possibly with mismatching versions (which is another problem...).